src/test/java/com/mcneilio/orcmaker/BasicEventDriver2Test.java

```diff
@@ -0,0 +1,85 @@
+package com.mcneilio.orcmaker;
+
+import com.mcneilio.orcmaker.storage.LocalStorageDriver;
+import org.apache.hadoop.hive.ql.exec.vector.BytesColumnVector;
+import org.apache.hadoop.hive.ql.exec.vector.LongColumnVector;
+import org.apache.orc.TypeDescription;
+import org.json.JSONObject;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.Properties;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+class BasicEventDriver2Test {
+
+  private BasicEventDriver2 driver;
+  private TypeDescription schema;
+
+  @BeforeEach
+  void setUp() {
+    Properties config = new Properties();
+    config.setProperty("event.name", "test_event");
+    config.setProperty("event.date", "2023-08-27");
+    config.setProperty("storage.driver", "local");
+    config.setProperty("storage.local.path", "/tmp/orc");
+
+    schema = TypeDescription.fromString(
+      "struct<name:string,flag:boolean,bignum:bigint,num:int,tinynum:tinyint,date:date>"
+    );
+
+    LocalStorageDriver storageDriver = new LocalStorageDriver(config);
+
+    driver = new BasicEventDriver2(config, schema, storageDriver, null);
```

Comment:
An example of how to do this basic config with the `BasicEventDriver` is available here: https://github.com/mcneiljt/shokuyoku/pull/23/files#diff-d03c15cfd5d59f8cda0a7b9c132b087ee5234e5bfa169c62696ae7d2b9bcbb0cR30-R40